bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh

```diff
@@ -96,7 +96,7 @@ prepare_reweight () {
 	mkdir -p madevent/Events/pilotrun
         cp $WORKDIR/unweighted_events.lhe.gz madevent/Events/pilotrun
         cd madevent
-        config=./madevent/Cards/me5_configuration.txt
+        config=./Cards/me5_configuration.txt # already in madevent dir
```

Comment:
To avoid such mistakes, wouldn't it make sense to give full path here?

Reply:
Valid point, will modify the commit (given that the Powheg+JHUGen bug is fixed)